schemas/com.snowplowanalytics.snowplow.badrows/enrichment_failures/jsonschema/1-0-0

```diff
@@ -0,0 +1,311 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a bad row resulting from enrichment failures",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "enrichment_failures",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the enrichment failures",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the failure occurred",
+          "format": "date-time"
+        },
+        "messages": {
+          "type": "array",
+          "description": "List of failure messages associated with the enrichment failures",
+          "items": {
+            "type": "object",
+            "description": "Individual enrichment failure",
+            "properties": {
+              "enrichment": {
+                "type": "object",
+                "description": "Information needed to identify an enrichment",
+                "properties": {
+                  "vendor": {
+                    "type": "string",
+                    "description": "Vendor of the enrichment which produced this failure",
+                    "pattern": "^[a-zA-Z0-9-_.]+$"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "Name of the enrichment which produced this failure",
+                    "pattern": "^[a-zA-Z0-9-_]+$"
+                  },
+                  "format": {
+                    "type": "string",
+                    "description": "Format of the enrichment which produced this failure",
+                    "pattern": "^[a-zA-Z0-9-_]+$"
+                  },
+                  "version": {
+                    "type": "string",
+                    "description": "Version of the enrichment which produced this failure",
+                    "pattern": "^[0-9]+-[0-9]+-[0-9]+$"
+                  },
+                  "identifier": {
```

Comment:
What we're going to put into `identified` until we don't a separate id for it?

Reply:
I don't know yet this is something to figure out or make not required

Reply:
I'd go with duplicated `$vendor/$name/jsonschema/$version` string until we have proper id. Because making the field required means it will be a new MODEL.